core/src/main/java/io/questdb/cutlass/http/processors/JsonQueryProcessor.java

```diff
@@ -264,7 +264,7 @@ protected static void header(
     }
 
     private static void readyForNextRequest(HttpConnectionContext context) {
-        LOG.debug().$("all sent [fd=").$(context.getFd()).$(']').$();
+        LOG.info().$("Request complete, ").$(context.getTotalBytesSent()).$(" bytes sent [fd=").$(context.getFd()).$(']').$();
```

Comment:
- it would be useful to track network usage per connection to try to find offenders. When total usage goes up, what do we do?
- log structure is "action done/doing [context_variable=value, context_variable2=value, ....]"